health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/Switch.js

```
@@ -0,0 +1,62 @@
+      setIsChecked(!isChecked);
+      onToggle(!isChecked);
+    }
+  };
+
+  return (
+    <div className={`digit-switch-container ${className || ""} ${disable ? "switch-disabled" : ""}`} style={style || {}}>
+      {isLabelFirst && <span className={`digit-switch-label ${"labelFirst"}`}>{label}</span>}
+      <div
+        className={`digit-switch ${isChecked ? "checked" : ""} ${shapeOnOff ? "shape-onoff" : ""} ${disable ? "switch-disabled" : ""}`}
+        onClick={handleToggle}
+        tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)